nautobot/dcim/views.py

```diff
@@ -2367,31 +2374,15 @@ class ModuleUIViewSet(BulkComponentCreateUIViewSetMixin, NautobotUIViewSet):
     table_class = tables.ModuleTable
     component_model = None
 
-    def get_action(self):
-        if self.component_model:
-            method = self.request.method.lower()
-            if method == "get":
-                return "view"
-            else:
-                return "change"
-
-        return super().get_action()
-
     def get_required_permission(self):
-        # TODO: standardize a pattern for permissions enforcement on custom actions
-        if self.component_model:
-            model = self.component_model
-            method = self.request.method.lower()
-            if method == "get":
-                component_action = "view"
-                permissions = [*self.get_permissions_for_model(model, [component_action]), "dcim.view_module"]
-            elif self.action.startswith("bulk_add"):
-                component_action = "add"
-                permissions = [*self.get_permissions_for_model(model, [component_action]), "dcim.change_module"]
-            else:
-                component_action = "change"
-                permissions = [*self.get_permissions_for_model(model, [component_action]), "dcim.change_module"]
-
+        # TODO figure out a case where we can handle this with custom_view_base_action and custom_view_additional_permissions attributes
+        # I am not too sure what views are using these permissions
```

Comment:
Looks like the existing logic was distinguishing between GET and POST methods to determine the appropriate permissions for the custom actions. Might need to restore that, might not?

Reply:
I think the implementation of `custom_view_base_action` eliminated the need for distinguishing between GET and POST requests.